README.md

```diff
@@ -21,7 +21,6 @@ Major tools/libraries:
 * RSpec and Capybara
 * Factory Bot
 * SimpleCov
-* Guard (rspec, livereload)
```

Comment:
We removed guard from the app template. See https://github.com/carbonfive/raygun-rails/pull/564